admin/class-aioseop-helper.php

```diff
@@ -129,300 +129,239 @@ private function help_text_general() {
+			'aiosp_rewrite_titles'              => __( "If enabled, all of your title and meta description tags will be rewritten, regardless of whether they were automatically generated by the plugin or entered by you manually. You can specify the title format for each post type or taxonomy below, as well as the description format.<br/><br/>For instance, all post titles will be rewritten as 'Post Title | Site Name' by default if you have this setting enabled. You can adjust each title format with your own text or by using our macros.", 'all-in-one-seo-pack' ),
 			'aiosp_home_page_title_format'      =>
-				__( 'This controls the format of the title tag for your Home Page.', 'all-in-one-seo-pack' ) . '<br />' .
+				__( 'This controls the format of the title tag for your Homepage.', 'all-in-one-seo-pack' ) . '<br />' .
 				__( 'The following macros are supported:', 'all-in-one-seo-pack' ) .
-				'<ul>' .
-					'<li>' .
-						__( '%site_title% - Your site title', 'all-in-one-seo-pack' ) .
-					'</li>' .
-					'<li>' .
-						__( '%site_description% - Your site description', 'all-in-one-seo-pack' ) .
-					'</li>' .
-					'<li>' .
-						__( '%page_title% - The original title of the page', 'all-in-one-seo-pack' ) .
-					'</li>' .
-					'<li>' .
-						__( "%page_author_login% - This page's author' login", 'all-in-one-seo-pack' ) .
-					'</li>' .
-					'<li>' .
-						__( "%page_author_nicename% - This page's author' nicename", 'all-in-one-seo-pack' ) .
-					'</li>' .
-					'<li>' .
-						__( "%page_author_firstname% - This page's author' first name (capitalized)", 'all-in-one-seo-pack' ) .
-					'</li>' .
-					'<li>' .
-						__( "%page_author_lastname% - This page's author' last name (capitalized)", 'all-in-one-seo-pack' ) .
-					'</li>' .
-					'<li>' .
-						__( '%current_date% - The current date (localized)', 'all-in-one-seo-pack' ) .
-					'</li>' .
-					'<li>' .
-						__( '%current_year% - The current year', 'all-in-one-seo-pack' ) .
-					'</li>' .
-					'<li>' .
-						__( '%cf_fieldname% - Custom field name', 'all-in-one-seo-pack' ) .
-					'</li>' .
-				'</ul>',
+				'<dl>' .
+					'<dt>%site_title%</dt>' .
+					'<dd>' . __( 'Your site title', 'all-in-one-seo-pack' ) . '</dd>' .
+					'<dt>%site_description%</dt>' .
+					'<dd>' . __( 'Your site description', 'all-in-one-seo-pack' ) . '</dd>' .
+					'<dt>%page_title%</dt>' .
+					/* translators: %s is replaced with a content type such as Post, Page, etc. */
+					'<dd>' . sprintf( __( 'The original title of the %s', 'all-in-one-seo-pack' ), __( 'Homepage', 'all-in-one-seo-pack' ) ) . '</dd>' .
+					'<dt>%page_author_login%</dt>' .
+					/* translators: Example sentence: "The first name of the author of the Post" */
+					'<dd>' . sprintf( __( 'The %1$s of the author of the %2$s', 'all-in-one-seo-pack' ), __( 'username', 'all-in-one-seo-pack' ), __( 'Homepage', 'all-in-one-seo-pack' ) ) . '</dd>' .
```

Comment:
These should probably be new lines since they extend too far; I think 120 is usually the character limit